setup.py

```diff
@@ -119,6 +124,29 @@ def _read(file):
 FREETYPE_ROOT = None
 LCMS_ROOT = None
 
+# ask pkg-config
+if _cmd_exists('pkg-config'):
+    roots = dict(
+        JPEG_ROOT="libjpeg",
+        JPEG2K_ROOT="libopenjp2",
+        TIFF_ROOT="libtiff-4",
```

Comment:
should i make it first search for ``libtiff-5`` and then ``libtiff-4``? or is ``libtiff-4`` preferred as a default?